docs/mp/config/01_introduction.adoc

```diff
@@ -21,15 +22,52 @@
 :toc-placement: preamble
 :spec-name: MicroProfile Config
 :description: {spec-name} support in Helidon MP
-:keywords: helidon, mp, microprofile, config
+:keywords: helidon, mp, microprofile, config, encryption, reference
+
 
 
 == About {spec-name}
+Helidon MP Config is an implementation of https://github.com/eclipse/microprofile-config/[Eclipse MicroProfile Config] that now includes Helidon-specific meta-configuration and APIs. You can configure your applications using MicroProfile's config properties file and APIs and extend the configuration system with Helidon-specific meta-configuration and APIs. You can also write and register custom ConfigSources.
```

Comment:
This does not read very nice.

1. the `now` in the first sentence is not correct. We already had support for that in 1.4
2. I would use the Oxford comma: "You can configure your application using MicroProfile's config properties and APIs, and extend the configuration system with Helidon specific features."
3. The last sentence is redundant (or not correct) - custom ConfigSources are part of the MicroProfile specification. It is usually called an SPI. Maybe a new paragraph about extensibility? E.g. "You can extend configuration with MicroProfile Config SPI for `ConfigSource` and `Converter` functionality."